spec/services/bulk_upload/lettings/year2023/row_parser_spec.rb

```diff
@@ -334,6 +334,276 @@
         end
       end
 
+      context "when valid row with valid decimal (integer) field_5" do
+        before do
+          allow(FeatureToggle).to receive(:bulk_upload_duplicate_log_check_enabled?).and_return(true)
+        end
+
+        let(:attributes) do
```

Comment:
Would it make sense to extract this into :valid_attributes and only edit relevant fields in the tests? Similar to what we have in some other parser tests

Reply:
yep good shout, will do :)